dhctl/pkg/server/rpc/dhctl/check.go

```diff
@@ -54,10 +51,15 @@ func (s *Service) Check(server pb.DHCTL_CheckServer) error {
 	doneCh := make(chan struct{})
 	internalErrCh := make(chan error)
 	receiveCh := make(chan *pb.CheckRequest)
-	sendCh := make(chan *pb.CheckResponse, 100)
+	sendCh := make(chan *pb.CheckResponse)
```

Comment:
There was channel with 100-capacity, and now it is 1-capactiy. Is it by intention or typo?

Reply:
The number 100 was added for debugging purposes a while ago, and I forgot to change it back afterward. It should indeed be 0-capacity as intended. As in another tasks. Sorry for the oversight!